Project.toml

```
@@ -1,7 +1,7 @@
 name = "AdvancedPS"
 uuid = "576499cb-2369-40b2-a588-c64705576edc"
 authors = ["TuringLang"]
-version = "0.3.0"
+version = "0.3.4"
```

```suggestion
version = "0.4"
```
the `master` branch contains breaking changes that have not been released yet. The most recent `v0.3x` releases are based on the [`releases` ](https://github.com/TuringLang/AdvancedPS.jl/tree/releases) branch.